mautrix_telegram/portal.py

```diff
@@ -2189,7 +2194,25 @@ def _parse_telegram_document_attributes(attributes: list[TypeDocumentAttribute])
                 width, height = attr.w, attr.h
             elif isinstance(attr, DocumentAttributeImageSize):
                 width, height = attr.w, attr.h
-        return DocAttrs(name, mime_type, is_sticker, sticker_alt, width, height, is_gif)
+            elif isinstance(attr, DocumentAttributeAudio):
+                is_audio = True
+                is_voice = attr.voice or False
+                duration = attr.duration
+                waveform = attr.waveform or bytes()
```

Comment:
```suggestion
                waveform = decode_waveform(attr.waveform) if attr.waveform else b""
```

(using `from telethon.utils import decode_waveform`)

Reply:
It seems like the waveform is already decoded (it's `bytes` or `None`). What does the `decode` do for us?

Reply:
It seems to decode from Telegram's weird 5-bit format into normal bytes